examples/plotting/file/airports_map.py

```diff
@@ -22,13 +17,14 @@
 tile_source = WMTSTileSource(**tile_options)
 
 # set to roughly extent of points
-x_range = Range1d(start=airports['x'].min() - 10000, end=airports['x'].max() + 10000)
-y_range = Range1d(start=airports['y'].min() - 10000, end=airports['y'].max() + 10000)
+x_range = Range1d(start=airports['x'].min() - 10000, end=airports['x'].max() + 10000, bounds=None)
+y_range = Range1d(start=airports['y'].min() - 10000, end=airports['y'].max() + 10000, bounds=None)
```

Comment:
I believe this is just a workaround to a bug. With this tile renderer the plot tools behave badly in conjunction with bounds. 

> On Jan 3, 2016, at 10:02, Sarah Bird <notifications@github.com> wrote:
> 
> In examples/plotting/file/airports_map.py:
> 
> > @@ -22,13 +17,14 @@
> >  tile_source = WMTSTileSource(**tile_options)
> >  
> >  # set to roughly extent of points
> > -x_range = Range1d(start=airports['x'].min() - 10000, end=airports['x'].max() + 10000)
> > -y_range = Range1d(start=airports['y'].min() - 10000, end=airports['y'].max() + 10000)
> > +x_range = Range1d(start=airports['x'].min() - 10000, end=airports['x'].max() + 10000, bounds=None)
> > +y_range = Range1d(start=airports['y'].min() - 10000, end=airports['y'].max() + 10000, bounds=None)
> I guess my question would be why would you want to pan far beyond the edge of the data? (Apart from the fact that this is a demo about the tile renderer) I have been assuming that having something that prevents you panning beyond the edges of your data is useful. This does raise an interesting idea for a feature though to express bounds as a multiple. Anyway, it's fixed for now. Happy to open a new issue to discuss.
> 
> —
> Reply to this email directly or view it on GitHub.
>